.ruby-version

```diff
@@ -0,0 +1 @@
+2.5
```

Comment:
I'd prefer not to commit this file to the repo because it can cause issues if you use the wrong Ruby version manager (chruby vs rbenv vs rvm). It can be set on CircleCI via:

```
      - run:
          name: Set Ruby Version
          command:  echo "ruby-2.5" > ~/.ruby-version
```

Reply:
Would you recommend doing this before every job? Do you happen to know a simple way to make this run for every job without having to repeat it?

Reply:
You can use a CircleCI 2.1 "command" to clean things up: https://github.com/Rightpoint/ios-template/blob/1b3a83cd06910cdf8795c5dea1e37682650fa7d8/.circleci/config.yml#L151

for example
```yml
version: 2.1

jobs:
  example:
    executor: xcode-10
    steps:
      - setup
      - etc...

commands:
  setup:
    description: "Shared setup"
    steps:
      - checkout
      - restore-gems

  restore-gems:
    description: "Restore Ruby Gems"
    steps:
      - run:
          name: Set Ruby Version
          command:  echo "ruby-2.5" > ~/.ruby-version
      - restore_cache:
          key: 1-gems-{{ checksum "Gemfile.lock" }}
      - run: bundle check || bundle install --path vendor/bundle
      - save_cache:
          key: 1-gems-{{ checksum "Gemfile.lock" }}
          paths:
            - vendor/bundle
```

Reply:
I usually check in `.ruby-version` to help ensure parity between local and CI environments. I think fully-qualified ruby versions should work between all three managers, but it is true that people must be careful not to use any that are specific to only one or two of them. Is there another way to ensure this? What issues have you seen?

Reply:
I specifically had an issue in the past where `chruby` expected `ruby-2.5` but `rbenv` wanted `2.5`, so it was impossible to support both. Maybe that's no longer an issue?